plugins/talk-plugin-featured-comments/client/index.js

```diff
@@ -84,8 +85,28 @@ export default {
           }
```

Comment:
Line 63-73 needs to be removed as it is replaced with the new `update` below.
https://github.com/coralproject/talk/pull/942/files#diff-8103bea6e6543f80c7f897579ce2dcbeL63